src/js/OTSession.coffee

```diff
@@ -134,13 +134,14 @@ class TBSession
       @resetElement(element)
       delete( streamElements[ elementId ] )
       TBUpdateObjects()
-    return Cordova.exec(TBSuccess, TBError, OTPlugin, "unsubscribe", [subscriber.streamId] )
+    return Cordova.exec(TBSuccess, TBError, OTPlugin, "unsubscribe", [subscriber.streamId, @sessionId] )
 
   constructor: (@apiKey, @sessionId) ->
     @apiKey = @apiKey.toString()
     @connections = {}
     @streams = {}
     @subscribers = {}
+    @capabilities = { publish: 1 }
```

Comment:
Mmm, should it always be 1 ?

Reply:
We are expecting 1 in the frontside
the web version returns capabilities, but the plugging doesn't so it's added arteficially